package.json

```diff
@@ -16,11 +16,14 @@
     "test": "ava",
     "dev": "netlify dev"
   },
+  "engines": {
+    "node": ">=18.0.0"
+  },
```

Comment:
ipx 2 uses H3, which uses global fetch objects. Rather than polyfilling it seems to make sense to just set the ngines requirement, as both Next and Gatsby now require it too